test/python/opflow/test_gradients.py

```diff
@@ -479,7 +476,7 @@ def test_state_hessian(self, method):
                 state_hess.assign_parameters(value_dict).eval(), correct_values[i], decimal=1
             )
 
-    @unittest.skipIf(not _HAS_JAX, "Skipping test due to missing jax module.")
+    @unittest.skipIf(not optionals.HAS_JAX, "Skipping test due to missing jax module.")
```

Comment:
A potential follow-on, not something we should do in this pr. Is to add a skip test decorator to the lazy importer class. So we can just make this `@optionals.HAS_JAX.test_requires` or something

Reply:
I'm not 100% sure on that - it would couple the objects to our testing framework, and that's not ideal for something we're publicly exporting.

Reply:
Well I was thinking more just having the decorator raise `unittest.SkipTest` which is in stdlib and while it's specifically part of unittest framework I think all the python testing frameworks support it. But it's not critical and I agree a gray area of whether we want that as part of the api. We can discuss it later if we think it's something worthwhile